src/Framework/AspNetCoreAnalyzers/test/Verifiers/CSharpRouteHandlerCodeFixVerifier.cs

```diff
@@ -64,8 +64,8 @@ public RouteHandlerAnalyzerTest()
         {
             // We populate the ReferenceAssemblies used in the tests with the locally-built AspNetCore
             // assemblies that are referenced in a minimal app to ensure that there are no reference
-            // errors during the build.
-            ReferenceAssemblies = ReferenceAssemblies.Net.Net60.AddAssemblies(ImmutableArray.Create(
+            // errors during the build. The value used here should be updated on each TFM change.
```

Comment:
This won't be remembered the next time we update TFMs. Please add the relevant information to docs/UpdatingMajorVersionAndTFM.md

Reply:
Done.